src/lib.rs

```diff
@@ -104,6 +104,9 @@
 
 // Make sure all our public APIs have docs.
 #![warn(missing_docs)]
+// Add options below to allow/deny Clippy lints.
+// Deny unused qualifications.
+#![deny(unused_qualifications)]
```

Comment:
```suggestion
// Deny unused qualifications.
#![deny(unused_qualifications)]
// Add options below to allow/deny Clippy lints.
```

It's better I think.

Reply:
Please click resolve conversation after you fixed this in your commits :)